### ISSUE TITLE
Why stop_gradient here?

### ISSUE THREAD

**Issue by guotong1988** [google-research/uda#84]
![image](https://user-images.githubusercontent.com/4702353/83832211-af95b100-a71b-11ea-8c91-a373b82f68fe.png)

Thank you very much.

@qizhex

**Comment by airship-explorer** [comment]
> ![image](https://user-images.githubusercontent.com/4702353/83832211-af95b100-a71b-11ea-8c91-a373b82f68fe.png)
> 
> Thank you very much.
> 
> @qizhex

tf.stop_gradient aims to convert the _ori_log_probs_ into constant Tensor, prevents the contribution of  _ori_log_probs_ to be taken into account for computing gradients.
